crime-commons-mod-schemas/src/main/resources/schemas/evidence/apiUpdateIncomeEvidenceRequest.json

```diff
@@ -70,5 +70,5 @@
     }
   },
   "additionalProperties": false,
-  "required": ["magCourtOutcome", "metadata", "evidenceDueDate", "financialAssessmentId"]
+  "required": ["magCourtOutcome", "metadata", "financialAssessmentId"]
```

Comment:
Would we ever have an evidence requirement where we don't have a due date?

Reply:
In Matt DB where FINANCIAL_ASSESSMENTS.INCOME_EVIDENCE_DUE_DATE is nullable filed and i can see number of assessment don't have due date.  

When updating income evidence from UI, the incomeEvidenceDueDate is in null  applicationDTO for  Maat id: 6012770